tree/ntuple/v7/src/libdaos_mock/libdaos_mock.cxx

```diff
@@ -299,91 +276,101 @@ int daos_oclass_id2name(daos_oclass_id_t oc_id, char *name)
 
 ////////////////////////////////////////////////////////////////////////////////
 
-int daos_cont_create(daos_handle_t poh, const uuid_t uuid, daos_prop_t *cont_prop, daos_event_t *ev)
+int daos_cont_create(daos_handle_t poh, uuid_t uuid, daos_prop_t * /*cont_prop*/, daos_event_t * /*ev*/)
 {
-   (void)cont_prop;
-   (void)ev;
-
    auto pool = RDaosHandle::ToPointer<RDaosFakePool>(poh);
+
    if (!pool)
       return -DER_INVAL;
-   Uuid_t u;
-   std::copy_n(uuid, std::tuple_size<Uuid_t>::value, std::begin(u));
-   pool->CreateContainer(u);
+
+   pool->CreateContainer((label_t) reinterpret_cast<char *>(uuid));
```

Comment:
Construct a temporary `label_t` (aka `std::string`) instead.  Same for other similar uses below.
```suggestion
   pool->CreateContainer(label_t(reinterpret_cast<char *>(uuid)));
```